lib/config/virtual_container_host.go

```diff
@@ -124,6 +124,8 @@ type Container struct {
 	ContainerNameConvention string
 	// Permitted datastore URLs for container storage for this virtual container host
 	ContainerStores []url.URL `vic:"0.1" scope:"read-only" recurse:"depth=0"`
+	// HostSelctionRequired vic to select host if DRS is disabled & ROBO Adv. License
+	HostSelectionRequired bool `vic:"0.1" scope:"read-only"`
```

Comment:
I was wondering whether adding this field would require a future change to account for `vic-machine upgrade` in the code path(s) where we apply VCH/container configuration (similar to what https://github.com/vmware/vic/pull/6284 did). However, we'll most likely run feature/license checks and set `HostSelectionRequired` separately every time we run `vic-machine upgrade`, so I don't think the said code change would be required.